packages/flutter/test/widgets/framework_test.dart

```diff
@@ -486,24 +486,23 @@ void main() {
     expect(
       element.toStringDeep(),
       equalsIgnoringHashCodes(
-        '''Column([GlobalKey#00000]; renderObject: RenderFlex#00000)
-├Container()
-│└LimitedBox(maxWidth: 0.0; maxHeight: 0.0; renderObject: RenderLimitedBox#00000 relayoutBoundary=up1)
-│ └ConstrainedBox(BoxConstraints(biggest); renderObject: RenderConstrainedBox#00000 relayoutBoundary=up2)
-├Container([GlobalKey#00000])
-│└LimitedBox(maxWidth: 0.0; maxHeight: 0.0; renderObject: RenderLimitedBox#00000 relayoutBoundary=up1)
-│ └ConstrainedBox(BoxConstraints(biggest); renderObject: RenderConstrainedBox#00000 relayoutBoundary=up2)
-├Container()
-│└Container()
-│ └LimitedBox(maxWidth: 0.0; maxHeight: 0.0; renderObject: RenderLimitedBox#00000 relayoutBoundary=up1)
-│  └ConstrainedBox(BoxConstraints(biggest); renderObject: RenderConstrainedBox#00000 relayoutBoundary=up2)
-├Container([GlobalKey#00000])
-│└LimitedBox(maxWidth: 0.0; maxHeight: 0.0; renderObject: RenderLimitedBox#00000 relayoutBoundary=up1)
-│ └ConstrainedBox(BoxConstraints(biggest); renderObject: RenderConstrainedBox#00000 relayoutBoundary=up2)
-└Container()
- └LimitedBox(maxWidth: 0.0; maxHeight: 0.0; renderObject: RenderLimitedBox#00000 relayoutBoundary=up1)
-  └ConstrainedBox(BoxConstraints(biggest); renderObject: RenderConstrainedBox#00000 relayoutBoundary=up2)
-''',
+        'Column([GlobalKey#00000]; renderObject: RenderFlex#00000)\n'
```

Comment:
I could go either way on this. Let's try to live with your change and see if the inconsistency is made up for by the cleaner output.